pkg/csi_driver/multishare_controller.go

```diff
@@ -573,6 +573,8 @@ func (m *MultishareController) generateNewMultishareInstance(instanceName string
 		// Ignore the cidr flag as it is not passed to the cloud provider
 		// It will be used to get unreserved IP in the reserveIPV4Range function
 		// ignore IPRange flag as it will be handled at the same place as cidr
+		case ParamNfsExportOptions:
```

Comment:
put this above the IP range comments and consider adding comments on why this is ignored